src/main/java/seedu/address/commons/util/DateTimeUtil.java

```diff
@@ -4,29 +4,33 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
 
-public class DateUtil {
+public class DateTimeUtil {
     public static final String DATE_CONSTRAINTS =
             "Date should be a valid date in the format of dd-MM-yyyy.";
     public static final String TIME_CONSTRAINTS =
             "Time should be a valid time in the format of HH:mm.";
     public static final String DATE_TIME_CONSTRAINTS =
             "DateTime should be a valid date and time in the format of dd-MM-yyyy HH:mm.";
-    public static final String DATE_VALIDATION_REGEX = "^(3[01]|[12][0-9]|0[1-9])-(1[0-2]|0[1-9])-[0-9]{4}$";
-    public static final String DATETIME_VALIDATION_REGEX =
-            "^(3[01]|[12][0-9]|0[1-9])-(1[0-2]|0[1-9])-[0-9]{4} (2[0-3]|[01][0-9]):([0-5][0-9])$";
-    public static final String TIME_VALIDATION_REGEX = "^(2[0-3]|[01][0-9]):([0-5][0-9])$";
-    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy");
-    public static final DateTimeFormatter DATETIME_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
-    public static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+    public static final String DATE_FORMAT = "dd-MM-yyyy";
+    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-uuuu")
```

Comment:
shouldn't it be 
public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern(DATE_FORMAT)

Reply:
I have removed DATE_FORMAT. Here the correct pattern for java DateTimeFormatter is actually "dd-MM-uuuu" instead of "dd-MM-yyyy".